### PR TITLE
First and second task

### PR DESCRIPTION
**Pull request by p-nedelchev**
Needs review

### CODE REVIEW

File: src/main/java/com/clouway/taskone/XmlCodec.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        xstream.processAnnotations(clazz);
+        return (T) xstream.fromXML(xml);
+    }
+
+    @Override
+    public <T> T unmarchall(String message, Type type) {
+        return (T) xstream.fromXML(message);
+    }
+
+    public Object unmarchall (String xml) {
```
Comment (tsony-tsonev) [inline src/main/java/com/clouway/taskone/XmlCodec.java]: what is the purpose of this method

File: src/test/java/JsonCodecTests.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    @Test
+    public void fromJson() throws Exception {
+        String name = "Paul";
+        String lastName = "Adams";
+        int age = 14;
+        Person expected = new Person(name, lastName, age);
+
+        String json = "{\"firstName\":\"" + name + "\",\"lastName\":\"" + lastName + "\",\"age\":" + age + "}";
+        Person actual = (Person) codec.unmarchall(json, Person.class );
```
Comment (tsony-tsonev) [inline src/test/java/JsonCodecTests.java]: are you sure you need this cast

**Comment by tsony-tsonev** [comment]
Don't forget to make the comparison between xml and json in the body of the PR.

**Comment by p-nedelchev** [comment]
|    Criteria   |              XML          |             JSON         |
| :-----------:|:--------------------:|:--------------------:|
| Size of file | Bigger than JSON  | Smaller                  |
| Speed       |  Slow                     | Faster than XML     |
| Readability| Harder to read cause of the tags confusing | Easy to read |

File: src/main/java/com/clouway/taskone/XmlCodec.java (hunk 1 above)
Comment (p-nedelchev) [inline src/main/java/com/clouway/taskone/XmlCodec.java]: I missed to remove it. I will fix it when I go to work.

File: src/test/java/JsonCodecTests.java (hunk 2 above)
Comment (p-nedelchev) [inline src/test/java/JsonCodecTests.java]: Actually IDEA proposed to put it there I will check again.

**Comment by tsony-tsonev** [comment]
OK LGTM when you fix these last comments

**Pull request merged**